copilot/fastapi/tests/conftest.py

```
@@ -0,0 +1,62 @@
+        mock_client.return_value = kubernetes_client
+        yield kubernetes_client
+
+@pytest.fixture
+def mock_argo_cd_token():
+    """Fixture to mock Argo CD authentication token"""
+    with patch("app.argo_cd_api.get_argo_cd_token") as mock_token:
+        mock_token.return_value = "test-argo-cd-token"
+        yield mock_token
+
+@pytest.fixture
+def mock_git_commands():
```

Ruff (0.8.2)

Missing return type annotation for public function `mock_git_commands`

(ANN201)